### ISSUE TITLE
Framerate drops when capturing

### ISSUE THREAD

**Issue by GhostSystemV2** [ArduCAM/ArduCAM_USB_Camera_Shield#93]
Hi! I'm having a problem that when i start capturing frames with the Streaming Demo Code, my frames drop from 54 to 11 or 9 fps, so i would like to know if the problem is with the camera or with the code.

I'm using an AR0134+USB3.0 Rev.C shield. 

![imagen](https://user-images.githubusercontent.com/13134854/52315336-22eaaf80-297c-11e9-95db-a556e6ed0cc5.png)

**Comment by ArduCAM** [comment]
Would you please let me know your hardware and software setup?

**Comment by GhostSystemV2** [comment]
Of course:

PC: 
Win10 x64
i7-8750H
32GB RAM

Camera:
Arducam AR0134 Color Camera Module
Arducam USB3.0 Camera Module REV.C

Software:
Python 3.6.8

The camera is conected to the pc to a usb3.0 port, the Streaming demo program is executed directly from the windows shell.

**Comment by ArduCAM** [comment]
Thanks for your info. Can you try our windows GUI from the github?  It is well tested and has more debugging options.
Few cases will case the frame drop.
1. Wrong connection of the USB port (not in your case), the USB2 and USB3 port has different settings for the frame rate
2. Auto exposure feature will automatically adjust the frame rate to promise the correct exposure (doesn't seems like your case)
3. Python script might degrade the performace when multi-threads racing

**Comment by GhostSystemV2** [comment]
Yes, the GUI version works fine with the camera, it doesn't drops frames.

So then, if the Python script is degrading the performance, the CPP code may get a better frame rate?

**Comment by ArduCAM** [comment]
Yes, you can try the CPP version as well. 
And please double check the config files if there is any difference between differernt versions.

**Comment by GhostSystemV2** [comment]
Okay, of course.

The "Coarse_integration_time R0x3012" could make the difference between the config files? i have seen some configuration files with a value of 10, some with 100, and even others with a number like 576.

**Comment by ArduCAM** [comment]
Let me know if you made any progress.

**Comment by GhostSystemV2** [comment]
Hi! 
i'm trying to implement the multiprocessing library, to get a better performance in the code but i get the next error:

![imagen](https://user-images.githubusercontent.com/13134854/52578326-a166a780-2de9-11e9-9224-56bf349cb2c6.png)

Any idea on how to solve it?

**Comment by ArduCAM** [comment]
Can you send me your code to support@arducam.com , then we can debug your code here.
